test/system/aws/logCollectionScript.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+ACCESSKEY=${1:-null}
+SECRETKEY=${2:-null}
+RUNNAME=${3:-null}
+COMMIT=${4:-null}
+host="$(hostname)"
+
+####copy all container logs in a given host.
```

Comment:
Do we copy the BK, hdfs logs too?

Reply:
No, we dont, till now we have never done that.